jetty-server/src/main/java/org/eclipse/jetty/server/handler/IPAccessHandler.java

```diff
@@ -201,19 +243,23 @@ public void setWhiteListByPath(boolean whiteListByPath)
     @Override
     public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException
     {
-        // Get the real remote IP (not the one set by the forwarded headers (which may be forged))
         HttpChannel channel = baseRequest.getHttpChannel();
-        if (channel!=null)
-        {
-            EndPoint endp=channel.getEndPoint();
-            if (endp!=null)
+        // If there are connectorNames specified, only apply this handler if this request is coming through one of
+        // those connectors.
+        if (_connectorNames.isEmpty() || _connectorNames.contains(channel.getConnector().getName())) {
```

Comment:
This should probably be a method along the lines of `isConnectorAllowed(Connector)`, similar to how `isAddrUriAllowed()` is below.

Reply:
good call